sdks/builds/Makefile

```diff
@@ -40,10 +41,7 @@ $(TOP)/tools/offsets-tool/MonoAotOffsetsDumper.exe: $(wildcard $(TOP)/tools/offs
 TARGETS=
 
 ## MXE targets
-ifndef DISABLE_ANDROID
```

Comment:
Why removing that? It's only useful for Android.

Reply:
So it can be tested without having an android sdk.

Reply:
If you don't define `DISABLE_ANDROID`, then you can just run `make provision-mxe`.

Reply:
Its not going to define MXE_PREFIX if its not included.

Reply:
I am sorry, I don't understand the issue you are running into. If you define `DISABLE_ANDROID`, then neither mxe.mk neither android.mk are included, so you don't need `MXE_PREFIX` anywhere. If you don't define `DISABLE_ANDROID`, then both mxe.mk and android.mk are included, and `MXE_PREFIX` is defined. Also, in android.mk, `$(MXE_PREFIX)` is going to be lazily evaluated (`=` vs `:=` for assignment). Finally, it's working as-is and I don't see where you are using `MXE_PREFIX` any differently now.